executor/aggregate_test.go

```diff
@@ -384,6 +384,9 @@ func (s *testSuite1) TestGroupConcatAggr(c *C) {
 	// issue #5411
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
+	// `group_concat` parallel gets chunks from blew `tableReader` and parallel running, the final result is unstable order in one group and hard to assert.
+	//  so set a big enough `InitChunkSize` to let tableReader return one group's result in one chunk to keep order.
+	tk.Se.GetSessionVars().InitChunkSize = 3
```

Comment:
how about setting it to `1` and then check the sorted result like this:
```go
result.Sort().Check(...)
```

With this, the test result can always be stable.